src/main/resources/anniv5Resources/localization/eng/secretlevelpack/Cardstrings.json

```diff
@@ -0,0 +1,46 @@
+{
+  "${ModID}:MysteryBox": {
+    "NAME": "Mystery Box",
+    "DESCRIPTION": "Apply !M! Vulnerable. NL Add a random Attack into your hand. NL If in a ? room, NL it costs 0 this turn."
+  },
+  "${ModID}:SurpriseAttack": {
+    "NAME": "Surprise Attack",
+    "DESCRIPTION": "Deal !D! damage. NL Apply !M! Weak. NL If it's turn 1, NL draw a card."
+  },
+  "${ModID}:EscapeClause": {
+    "NAME": "Escape Clause",
+    "DESCRIPTION": "Gain !B! Block. NL If any enemy has Weak or Vulnerable, play the top card of your draw pile."
+  },
+  "${ModID}:Banana": {
+    "NAME": "Banana",
+    "DESCRIPTION": "Heal !M! HP. NL Gain !${ModID}:m2! Dexterity. NL Put a *Trip on top of your draw pile."
+  },
+  "${ModID}:Joust": {
+    "NAME": "Joust",
+    "DESCRIPTION": "Can only target Weak enemies. NL Deal !D! damage. NL Apply !M! Weak.",
+    "EXTENDED_DESCRIPTION": [
+      "That foe is not Weak!"
+    ]
+  },
+  "${ModID}:Conglaturation": {
+    "NAME": "Conglaturation!",
+    "DESCRIPTION": "Deal !D! damage. NL If you played a glowing card this turn, gain [E] and draw 1 card."
```

Comment:
blue is also a glow

Reply:
Clarified; changed those cases to "gold-glowing"